tensorrt_llm/executor/rpc_proxy.py

```
@@ -276,29 +284,48 @@ def shutdown(self):
         # 3. shutdown the mpi session, this should wait until all the PyExecutor
         # processes are shutdown
         if self.mpi_session is not None:
-            logger_debug(f"Shutting down mpi session", color="yellow")
-            self.mpi_session.shutdown()
+            if self._owns_mpi_session:
+                logger_debug(f"Shutting down mpi session", color="yellow")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)